src/md.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const path = require('path');
+
+const mdLinks = (pathDocument) => new Promise((resolve, err) => {
+
+  const toAbsolutePath = path.resolve(pathDocument);
+  const docExtension = path.extname(toAbsolutePath);
+  /* const linksRegExp = /\[((.+?))\]\((http|https).+?\)/g;
+  const hrefRegExp = /\((http|https).+?\)/g; */
+  const textLinksRegExp = /\[.+?\]/g;
+
+  if (docExtension === '.md') {
+
+    const docMdToString = fs.readFileSync(toAbsolutePath).toString();
+    const linksMatchInMd = docMdToString.match(linksRegExp);
```

Comment:
esta genial el nombre de tus variables, te recomiendo que utilices let para cuando el valor muta.
```suggestion
    let docMdToString = fs.readFileSync(toAbsolutePath).toString();
    let linksMatchInMd = docMdToString.match(linksRegExp);
```